roles/iridl/templates/generate-forecasts

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I would rather not make silly changes just to placate ansible-test. I don't know that it has ever found a true mistake yet, so if we have to do crap like this it might not be worth it.

That said, according to the [docs](https://docs.ansible.com/ansible-core/2.18/dev_guide/testing/sanity/shebang.html) `#!/bin/bash -eu` is one of the acceptable shebangs. This script already uses `set -e` on line 3, and `-u`  seems like a good idea in general, so let's go with that.